### PR TITLE
Added message for blocked account on visaCal

### PR DESCRIPTION
**Pull request by talsalmona**
After a few failed login attempts, Visa Cal will ask you to change your password.
This will spit out a message for this use case instead of an unknown error.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "npm": ">= 5.3.0"
   },
   "main": "lib/index.js",
+  "publishConfig": { "registry": "https://npm.pkg.github.com/" },
```
Comment (eshaham) [inline package.json:10]: @talsalmona no need for this change

File: src/constants.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ export const LOGIN_RESULT = {
   SUCCESS: 'SUCCESS',
   INVALID_PASSWORD: 'INVALID_PASSWORD',
   CHANGE_PASSWORD: 'CHANGE_PASSWORD',
+  BLOCKED_USER: 'BLOCKED_USER',
```
Comment (eshaham) [inline src/constants.js:27]: @talsalmona sorry for being picky, I think that `USER_BLOCKED` works better as a login result in this case

**Comment by esakal** [comment]
@talsalmona Hello :)

In readme.md file there is a place that list the available results and in contributing.md there are few.

can you please add the new option to that list, thanks!

File: package.json (hunk 1 above)
Comment (talsalmona) [reply package.json:10]: Sorry, was unintentional. I'll remove it.

**Comment by esakal** [comment]
@talsalmona Hi, there are two more comments open and we will be able to merge it.
Thanks :)

**Comment by esakal** [comment]
@eshaham how do you want to proceed with this one?

**Comment by talsalmona** [comment]
Hi, sorry for the delay. Last week was very busy.
Please review my latest changes.
Also, I believe the README and CONTRIBUTING had an error where the possible return type on error we not correct.
WDYT?

**Comment by esakal** [comment]
@eshaham likewise :) please release a new official version when possible.

@talsalmona In the meanwhile feel free to pull the next tag from npm:

```
npm i israeli-bank-scrapers@next
```

**Pull request merged**